tools/rpkg/R/Result.R

```diff
@@ -67,6 +71,24 @@ list_to_df <- function(x) {
   x
 }
 
+list_to_tibble <- function(x) {
+  if (!requireNamespace("tibble", quietly = TRUE)) {
```

Comment:
Should the availability of the tibble package be checked when establishing the connection?